aclk/aclk_lws_wss_client.c

```diff
@@ -208,42 +208,81 @@ void aclk_lws_wss_client_destroy()
 #endif
 }
 
-int aclk_wss_set_socks(struct lws_vhost *vhost, const char *socks) {
+static int aclk_wss_set_socks(struct lws_vhost *vhost, const char *socks)
+{
     char *proxy = strstr(socks, ACLK_PROXY_PROTO_ADDR_SEPARATOR);
 
-    if(!proxy)
+    if (!proxy)
```

Comment:
There is unnecessary duplication in the code. This code path cannot be reached unless aclk_find_proxy found the appropriate prefix, then code is being run to check the prefix again.